firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -109,6 +102,13 @@ static bool startupPositionError = false;
 
 #define STARTUP_NEUTRAL_POSITION_ERROR_THRESHOLD 5
 
+static SensorType indexToTpsSensor(size_t index) {
+	switch(index) {
```

Comment:
code style - space is missing?

Reply:
you mean between `switch` and `(index)`?

Reply:
lol why'd you merge before I could fix this 🤣

Reply:
yes before "switch" and "("